scripts/no-push-files/README.md

```diff
@@ -47,10 +47,8 @@ dependencies:
 import 'package:countly_flutter/countly_flutter.dart';
 
 void main() {
-  // If you want to catch Dart errors, run your app inside a Zone and pass Countly.recordDartError as the onError parameter to it.
-  runZonedGuarded<void>(() {
-    runApp(MaterialApp(home: const MyApp()));
-  }, Countly.recordDartError);
+  // If you want to catch Dart errors, call enableCrashReporting before you initialize your Countly SDK.
```

Comment:
also remove this comment. if they want to capture any error the method is same. no need to single out dart

Reply:
Done